source/stylesheets/site.css.scss

```diff
@@ -247,6 +247,46 @@ body {
   margin-left: 65px;
 }
 
+.gridlines-wrapper {
+  position: absolute;
+  width: 100%;
+}
+
+.gridline {
+  position: absolute;
+  width: 1;
+  height: 500;
+  border: 0;
+  border-left: 2px solid orange;
+}
+
+.ductwork {
+  top: 47px;
+  width: 100%;
+  border: 0;
+  border-top: 2px solid orange;
+}
+
+.grid-col {
+  position: absolute;
+  height: 900px;
+  width: 61px;
+  border-left: 2px solid orange;
+  border-right: 2px solid orange;
```

Comment:
@claytongentry A good best practice with your CSS is to order the attributes alphabetically. For example, the border-left and border-right would go higher on the list and width goes at the bottom. Makes it easier to scan when you're deep in the CSS/SCSS.